src/plugins/legacy/polygonRoi/polygonEventFilter.cpp

```diff
@@ -937,6 +1101,27 @@ bool polygonEventFilter::isContourInSlice()
     return false;
 }
 
+bool polygonEventFilter::isOnlyOneNodeInSlice()
+{
+    bool res = false;
+    for (medTagRoiManager *manager : managers)
+    {
+        polygonRoi *roi = manager->existingRoiInSlice();
+        if (roi)
+        {
+            if (roi->getNumberOfNodes()==1)
+            {
+                res = true;
+            }
+            else if ( roi->getNumberOfNodes()>1)
```

Comment:
this `else if` is not needed since `res` is already false from line 1106

Reply:
nope.
If we find a roi with more than one node, we want to stop and quit.
We do not want to verify the other roi on other label